scripts/prepro_feats.py

```diff
@@ -50,6 +46,13 @@
 from misc.resnet_utils import myResnet
 import misc.resnet as resnet
 
+
+def append_array(f, f_toc, name, array):
+  offset = f.tell()
+  np.save(f, array)
+  f_toc.write("{} {}\n".format(name, offset))
```

Comment:
Forget about my deleted question. You can actually do `np.savez_compressed(f, feat=array)`.

Reply:
I do find np.savez_compressed will cause some weird bug, but I don't know why that happens.

Reply:
I think I found the solution.